backend/app/api/error_management.py

```
@@ -78,9 +89,8 @@ async def get_error_metrics(
         )
 
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)